doc/modules/compose.rst

```diff
@@ -244,11 +244,11 @@ object::
 Transforming target in regression
 =================================
 
-:class:`TransformedTargetRegressor` transforms the targets ``y`` before fitting
-a regression model. The predictions are mapped back to the original space via
-an inverse transform. It takes as an argument the regressor that will be used
-for prediction, and the transformer that will be applied to the target
-variable::
+:ref:`TransformedTargetRegressor <transformed_target_regressor>` transforms the
```

Comment:
Isn't this meant to be a link to the class?

Reply:
Yes it is, but it doesn't, and I don't know why. We could do an explicit 

    :class:`composeTransformedTargetRegressor`

instead. Do you prefer that?

Reply:
This will work:

```
:class:`~sklearn.compose.TransformedTargetRegressor` 
```

The full path is needed because the `currentmodule` is set to `sklearn.pipeline`. The `~` is to only display the `TransformedTargetRegressor` part in the output.